examples/subplots_axes_and_figures/subplots_adjust.py

```diff
@@ -1,10 +1,12 @@
 """
-===============
-Subplots Adjust
-===============
+=============================
+Subplots spacings and margins
+=============================
 
-Adjusting the spacing of margins and subplots using
-`~matplotlib.pyplot.subplots_adjust`.
+Adjusting the spacing of margins and subplots using `.pyplot.subplots_adjust`
+or `.pyplot.subplot_tool`.
```

Comment:
```suggestion
You can adjust the margins and spacings of subplots using `.pyplot.subplots_adjust`.

.. note::

    There's also a tool window to adjust the margins and spacings of displayed figures interactively. It can be opened via the toolbar or by calling `.pyplot.subplot_tool`.
```

(please line-wrap properly)

Reply:
agreed